docs/source/contrib_docs/doc-new-build.md

```diff
@@ -0,0 +1,33 @@
+# Building automatically on ReadTheDocs
```

Comment:
Not sure why this need to be explained there. The docs are already building, why would we need to set that up again >